tests/app/controller/command/commands/mention_test.py

```diff
@@ -20,30 +20,32 @@ def setUp(self):
 
     def test_handle_no_input(self):
         """Test handle command with no additional args."""
-        self.assertEqual(self.testcommand.handle('UFJ42EU67', user),
+        self.assertEqual(self.testcommand.handle(f"{user2}", user1),
                          ("invalid command", 200))
 
     def test_handle_wrong_input(self):
         """Test handle command with unsupported function."""
-        self.assertEqual(self.testcommand.handle('UFJ42EU67 --', user),
+        self.assertEqual(self.testcommand.handle(f"{user2} --", user1),
                          (self.testcommand.unsupported_error, 200))
 
     def test_handle_add_karma_to_another_user(self):
         """Test handle command with karma to another user."""
-        user = User('U123456789')
-        user.name = 'U123456789'
-        self.mock_facade.retrieve.return_value = user
-        self.assertEqual(self.testcommand.handle(('UFJ42EU67 ++'), user),
+        reciever = User('U123456789')
+        reciever.name = 'U123456789'
+        giver = 'UFJ42EU67'
+        self.mock_facade.retrieve.return_value = reciever
+        self.assertEqual(self.testcommand.handle("U123456789 ++", giver),
                          ("gave 1 karma to U123456789", 200))
-        self.mock_facade.retrieve.assert_called_once_with(User, "UFJ42EU67")
+        self.mock_facade.retrieve.assert_called_once_with(User, 'U123456789')
```

Comment:
Since you return the receiver object, you can make an assertion about the receiver's karma directly. Something like:

```python
self.assertEqual(receiver.karma, 2)
```